libsquoosh/src/emscripten-utils.ts

```diff
@@ -10,10 +10,14 @@ export function pathify(path: string): string {
 export function instantiateEmscriptenWasm<T extends EmscriptenWasm.Module>(
   factory: EmscriptenWasm.ModuleFactory<T>,
   path: string,
+  workerWasm: string = '',
 ): Promise<T> {
   return factory({
-    locateFile() {
-      return pathify(path);
+    locateFile(requestPath) {
+      if (requestPath.endsWith('.wasm')) return pathify(path);
```

Comment:
I'll admit I don't understand this logic / why we need different logic for different types of files here. Maybe a comment would be helpful?

Reply:
The build system for libSquoosh doesn’t understand `new URL(..., import.meta.url)` for assets yet, so we need to manually inject the updated paths here.

Reply:
I’ll add a comment